Trace.py

```diff
@@ -215,7 +215,7 @@ def run_fifo(self, put_target):
         self._write_put(put_target)
 
         # FIXME: we can avoid this read in some cases, as we should know where we are
-        self.real_dma_get_addr = self.xbox.read_u32(XboxHelper.DMA_GET_ADDR)
+        self.real_dma_get_addr = self.xbox.read_u32(XboxHelper.DMA_PULL_ADDR)
```

Comment:
If this is wrong, then the variable name on the LHS is also broken

Reply:
(marking resolved as this was fixed)